cmd/node/flags.go

```diff
@@ -507,26 +517,66 @@ func getWorkingDir(workingDir string, log logger.Logger) string {
 }
 
 func applyCompatibleConfigs(log logger.Logger, configs *config.Configs) error {
-	importDbFlags := configs.ImportDbConfig
-	importDbFlags.ImportDbNoSigCheckFlag = importDbFlags.ImportDbNoSigCheckFlag && importDbFlags.IsImportDBMode
-	importDbFlags.ImportDbSaveTrieEpochRootHash = importDbFlags.ImportDbSaveTrieEpochRootHash && importDbFlags.IsImportDBMode
+	if configs.FlagsConfig.EnablePprof {
+		runtime.SetMutexProfileFraction(5)
+	}
 
-	if importDbFlags.IsImportDBMode {
-		return processConfigImportDBMode(log, configs)
+	operationModes := strings.Split(configs.FlagsConfig.OperationMode, ",")
+	err := checkOperationModes(operationModes)
+	if err != nil {
+		return err
+	}
+
+	isInImportDBMode := configs.ImportDbConfig.IsImportDBMode || sliceContainsElement(operationModes, operationModeImportDb)
```

Comment:
I will not mix the import-db mode here as we specifically need to provide the DB path for the import operation

Reply:
the `import-db` mode is optional and a not breaking change, since the old flag is perfectly valid

Reply:
But it won't work if you just specify the operation mode.
Why? Because the `import-db` flag will hold the empty string, meaning that the source directory will be the same as the current directory and then the source directory will be equal to the destination directory leading to a failed operation.

Reply:
ok, removed the `import-db` from the operation modes list